openshift/README.md

```diff
@@ -0,0 +1,87 @@
+# OpenShift Cluster API Operator helm chart
+
+OpenShift (OCP) Cluster API Operator helm chart is based on the k8s-sig cluster-api-operator helm [chart](https://github.com/kubernetes-sigs/cluster-api-operator/tree/main/hack/charts/cluster-api-operator).  The OpenShift helm chart uses the [redhat-registry](https://catalog.redhat.com/software/containers/explore) container images to deploy the cluster-api-operator, cluster-api and cluster-api-aws-providers.
+
+## Install
+
+Add CAPI Operator & cert manager helm repository:
+
+```
+$ helm repo add capi-operator https://raw.githubusercontent.com/openshift/cluster-api-operator/refs/heads/main/openshift
+$ helm repo add jetstack https://charts.jetstack.io --force-update
+$ helm repo update
+```
+
+Install cert manager:
+
+```
+$ helm install cert-manager jetstack/cert-manager --namespace cert-manager --create-namespace --set installCRDs=true
+```
+
+Follow the instructions below to create the AWS credentials environment variable:
+
+```
+$ export AWS_REGION=us-east-1 
+$ export AWS_ACCESS_KEY_ID=<your-access-key>
+$ export AWS_SECRET_ACCESS_KEY=<your-secret-access-key>
+$ export AWS_SESSION_TOKEN=<session-token> # If you are using Multi-Factor Auth.
+$ export AWS_B64ENCODED_CREDENTIALS=$(clusterawsadm bootstrap credentials encode-as-profile|base64 -w0)
+$ echo $AWS_B64ENCODED_CREDENTIALS
+```
+
+Install the OpenShift Cluster-api-operator
+
+```
+$ helm upgrade --install capi-operator capi-operator/cluster-api-operator --create-namespace -n capi-operator-system --set awsEncodedCredentials=$AWS_B64ENCODED_CREDENTIALS
```

Comment:
what about a `--wait` parameter?

Reply:
not sure what you mean for --wait. Its an option that the user can use, We don't have to set this option. The chart should be installed and running without the need for this option.